course_discovery/urls.py

```diff
@@ -47,6 +47,7 @@
     url(r'^i18n/', include('django.conf.urls.i18n')),
     url(r'^jsi18n/$', JavaScriptCatalog, name='javascript-catalog'),
     url(r'^taggit_autosuggest/', include('taggit_autosuggest.urls')),
+    url(r'^edly_extensions/', include('course_discovery.apps.edly_extensions.urls', namespace='edly_extensions')),
```

Comment:
Please rename the app to `edx_discovery_app` and also modify the URLs:

```python
url(r'^edly_api/', include('course_discovery.apps.edx_discovery_app.urls', namespace='edly_api')),
```

CC: @iam-mhaseeb

Reply:
Done